components/wb-bubble/_base.scss

```diff
@@ -0,0 +1,110 @@
+
+/*
+ * Web Experience Toolkit (WET) / Boîte à outils de l'expérience Web (BOEW)
+ * wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * Bubble - Stable CSS
+ */
+
+.wb-bubble-wrap {
+	bottom: 30px;
+	height: 60px;
+	position: fixed;
+	right: 30px;
+	width: 60px;
+	z-index: 1049;
+	p {
+		background: #335075;
+		border-bottom-left-radius: 26px;
+		border-top-left-radius: 26px;
+		box-shadow: 0 1px 3px rgba(0, 0, 0, .45);
+		color: #fff;
+		font-size: .8em;
+		line-height: 1.5em;
+		min-height: 58px;
+		padding: 5px 37.5px 5px 27.5px;
+		position: relative;
+		right: 195px;
+		top: 1px;
+		width: 225px;
+
+		.notif-close {
+			background: #333;
+			border: none;
+			border-radius: 50%;
+			color: #fff;
+			font-size: 19px;
+			height: 1.25em;
+			line-height: 21px;
+			position: absolute;
+			right: 92.5%;
+			text-align: center;
+			text-decoration: none;
+			top: 0;
+			width: 1.25em;
```

Comment:
Is the button width and height are compatible with the WCAG 2.1 target size?

Reply:
@duboisp from my understanding, in order to meet WCAG 2.1 target size, the button needs to be at least 44px x 44px. The button generated by the bubble is 65px x 65px.